src/insights/funding.ts

```diff
@@ -0,0 +1,18 @@
+import { PageMeta, TimeMeta } from "./common";
+
+export interface FundingHistory {
+  lastFundingAt: string
+  market: string
+  fundingRate: string
+}
+
+export interface QueryGetFundingRateResponse {
+  entries: FundingHistory[]
+  meta: PageMeta 
```

Comment:
This `meta` struct does not look the same as `PageMeta` as it contains `from` and `until` and does not include count.
<img width="527" alt="Screenshot 2022-09-15 at 6 17 24 PM" src="https://user-images.githubusercontent.com/65583522/190379524-3db6fb04-cb85-47cf-834e-9b99fbeb6c47.png">
Would be good to declare a new struct altogether